### PR TITLE
add support oracle oci object storage

### PR DESCRIPTION
**Pull request by hymvp**
# Description

Add a new provider oracle oci object storage

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Tested in the local environment, the api directory was packaged and mirrored, and fully tested

# Suggested Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: api/extensions/storage/oci_storage.py
Comment (takatost) [inline api/extensions/storage/oci_storage.py]: Please use `app.config` to fetch configurations, and declare configuration parameters in the `configs`. You can refer to other storage implementations for guidance. Thank you!

**Review by takatost — CHANGES_REQUESTED** [review]

File: api/extensions/storage/oci_storage.py
Comment (hymvp) [reply api/extensions/storage/oci_storage.py]: hi @takatost ,i had modify it,please check it ,Thanks!

**Comment by hymvp** [comment]
hi @takatost ,I have two questions to consult.
1. Is the current status just waiting for approval? Is there anything I need to do?
2. When I modify docker-compose.yaml， add the oci configuration , a conflict will be prompted. How to solve it?
Thank you！

**Review by takatost — APPROVED** [review]
thx!

**Pull request merged**
